pages/events/index.vue

```diff
@@ -62,7 +62,13 @@
           />
         </div>
         <div v-else class="event-page__results">
-          <EmptyCard :title="emptyCardTitle" :description="emptyCardDescription" />
+          <AppBasicCard
+            class="event-page__no-events-card"
+            :image="emptyCard.img"
+            :title="emptyCard.title"
+          >
+            {{ emptyCard.description }}
```

Comment:
For this description (no events description), the design in Figma looks like it shouldn't go across the full width of the card - my initial guess would be a max-width of 26rem (calculated off design in Figma)

**Figma**
![Screen Shot 2020-10-01 at 8 10 07 AM](https://user-images.githubusercontent.com/6276074/94820897-33a43080-03be-11eb-9d1f-540c7ebeaa44.png)

**Preview in Vercel**
![Screen Shot 2020-10-01 at 8 16 00 AM](https://user-images.githubusercontent.com/6276074/94821030-5df5ee00-03be-11eb-9c8c-ad687d913016.png)


I also wish we could remove the darker gray borders above/below the no-events image. I'm fine if we save that for later since we're still working to improve event images altogether, with the Events team.

Is there a way for the the `__no-events-card` to just be that solid `$cool-gray-30`?

Reply:
After [this conversation](https://github.com/Qiskit/qiskit.org/pull/954#discussion_r495149641) we decided to have these darker gray borders when the img don't fit all the space, I think we should have a consistent design for all cards. Aren't you agree?

Reply:
That's true... we did have that conversation. I'm okay to let this in and fix later if needed.

Reply:
Agreed on the consistency.

Reply:
The dark background act as bands for filling images that do not accommodate to the container because we have no control on them. Fortunately, we can control everything regarding the empty card so I think these bands should not exist.

@JRussellHuffman what do you think?

Reply:
This has been resolved with the latest commits, thank you Yaiza.